src/main/java/nl/novi/TechItEasy/controllers/TelevisionController.java

```diff
@@ -0,0 +1,75 @@
+package nl.novi.TechItEasy.controllers;
+
+import nl.novi.TechItEasy.exceptions.RecordNotFoundException;
+import nl.novi.TechItEasy.services.Television;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+
+@RestController
+public class TelevisionController {
+    private ArrayList<Television> televisions;
+
+    public TelevisionController() {
+        televisions = new ArrayList<>();
+
+        Television t = new Television();
+        t.setName("XboxTV");
+        t.setBrand("Microsoft");
+        t.setPrice(499.99);
+        televisions.add(t);
+    }
+
+
+    @GetMapping("/televisions")
+    public ResponseEntity<Object> getAllTelevisions() {
+        if (televisions.size() > 0) {
+            return new ResponseEntity<>(televisions, HttpStatus.OK);
+        } else {
+            throw new RecordNotFoundException("No televisions found");
+        }
+    }
+
+    @GetMapping("/televisions/{id}")
+    public ResponseEntity<Object> getTelevision(@PathVariable int id) {
```

Comment:
als je een tv wilt targeten op int id zorg dan ook dat je in je Television class een 
private int id aanmaakt. Je gebruikt int id hier nu om een bepaalde index in je array te targeten terwijl je hem wilt gebruiken om de id van een specifiek object te matchen